package.json

```diff
@@ -7,8 +7,8 @@
     "JSON2": "0.1.0",
     "batch": "0.5.0",
     "bouncy": "3.2.2",
-    "browserify": "6.3.3",
-    "browserify-istanbul": "0.1.3",
+    "browserify": "8.1.3",
```

Comment:
I just did not wanted to lose time with outdated dependencies while using watchify. I know you do not like to update deps for the sake of updating, but waiting too much might do more harm than good?